src/layout/content/settings/settings-page.js

```diff
@@ -26,6 +28,7 @@ export class SettingsPage extends LitElement {
     this.autoplay = preferences.autoplay ?? false;
     this.muted = preferences.muted ?? true;
     this.debug = preferences.debug ?? false;
+    this.dataProviderHost = preferences.dataProviderHost || '';
```

Comment:
Does this mean that the default value of `this.dataProviderHost` can be an empty string? And therefore that this value can be passed to srgOptions.dataProviderHost? If so, this could cause unexpected behavior.

Reply:
I was saving this preference as an empty string when unset since an undefined value for the `input` field caused the placeholder to disappear. I've made it so that the value in the preference is saved as undefined but doesn't affect the input value.